### PR TITLE
Business/products page and add-product page

### PR DESCRIPTION
**Pull request by omotoshoayomikun**
Also, I merged the dev branch with my local branch and solved the search bar displaying on all screens.

### CODE REVIEW

File: src/app/[locale]/business/products/page.tsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    check === 'without_payment' &&
+      setChecked({ ...checked, without_payment: !checked.without_payment });
+  };
+
+  const handleAddBtn = () => {
+    router.push('/add-product')
+  }
+
+
+  return (
```
Comment (swoocn) [inline src/app/[locale]/business/products/page.tsx:45]: Hey @omotoshoayomikun - The Angular version of this configuration screen is actually in the business-settings/business-menu/business-menu.component.html and not the business-settings/products/products-settings/products-settings.component.html. 
There you will see the correct translated references.  Sorry for the confusion!

**Comment by swoocn** [comment]
Could you please move both screens to the /seller/registration folder?  Feel free to build new subfolders i.e., configuration, add-product.  Thanks!

File: src/app/[locale]/add-product/page.tsx
Comment (swoocn) [inline src/app/[locale]/add-product/page.tsx:1]: Could you please move this file to the [locale]/seller/registration/add-product folder structure instead?

File: src/app/[locale]/business/products/page.tsx
Comment (swoocn) [inline src/app/[locale]/business/products/page.tsx:1]: Could you please move this file to the [locale]/seller/registration/configuration folder structure instead?

File: src/app/[locale]/layout.tsx [hunk 2]
```diff
@@ -91,7 +90,6 @@ export default function LocaleLayout({
             {children}
           </Providers>
         </NextIntlClientProvider>
-        <SearchBar />
```
Comment (swoocn) [inline src/app/[locale]/layout.tsx:94]: 👍

File: src/app/[locale]/business/products/page.tsx (hunk 1 above)
Comment (swoocn) [reply src/app/[locale]/business/products/page.tsx:45]: I can send you the exact screenshot once the prod issue is resolved, but it's basically the same screen layout as what you have other than the toggling and button behavior.  For now, you can just copy over the corresponding translated content that exists in the /business-menu.component.html. i.e., BUSINESS.CONFIGURATION.LABELS.*  We can revisit the details of the screen later. 
Thanks!

Comment (omotoshoayomikun) [reply src/app/[locale]/business/products/page.tsx:45]: > I can send you the exact screenshot once the prod issue is resolved, but it's basically the same screen layout as what you have other than the toggling and button behavior. For now, you can just copy over the corresponding translated content that exists in the **/business-menu.component.html**. i.e., BUSINESS.CONFIGURATION.LABELS.* We can revisit the details of the screen later. Thanks!

Okay that's true but I think it has a pop up model so i will work on it.

**Review by swoocn — APPROVED** [review]
LGTM 😎👍

**Pull request merged**